configs/components/ruby-augeas.rb

```diff
@@ -24,6 +24,10 @@
       pkg.environment "RUBY" => settings[:host_ruby]
     end
     ruby = "#{settings[:host_ruby]} -r#{settings[:datadir]}/doc/rbconfig.rb"
+  elsif platform.is_huaweios?
+    pkg.environment "RUBY" => settings[:host_ruby]
+    ruby = "#{settings[:host_ruby]} -r#{settings[:datadir]}/doc/rbconfig.rb"
+    pkg.environment "LDFLAGS" => settings[:ldflags]
```

Comment:
We *should* but I'm not sure we do, I've sen this declared like this elsewhere too.